src/main/java/org/jboss/pnc/bifrost/endpoint/LogUpload.java

```diff
@@ -71,7 +80,7 @@ public String uploadFinalLog(@Valid FinalLogUpload logUpload, @Context HttpHeade
         finalLog.md5sum = logUpload.getMd5sum();
 
         if (logUpload.getTag() != null) {
-            finalLog.tags = Arrays.asList(logUpload.getTag().split(","));
+            finalLog.tags = Set.of(logUpload.getTag().split(","));
```

Comment:
Could you please update https://github.com/project-ncl/bifrost/blob/master/src/main/resources/db-update/00001-from-2.1.0-to2.1.1.sql with the changes for list to set? I think that's a good change (but I don't really know what Hibernate does with sets!)

Reply:
Oh just playing with it locally, looks like there are no SQL schema changes necessary right?

Reply:
So, I've gone through Hibernate docs and apparently there is no order column by default on `List`s as I've initially thought. So there shouldn't be a change in SQL scripts.